queries/ron/indents.scm

```diff
@@ -0,0 +1,12 @@
+[
+  (array)
+  (map)
+  (tuple)
+  (struct)
+] @indent
+
+[ "{" "}" ] @branch
+
+[ "(" ")" ] @branch
+
+[ "[" "]" ] @branch
```

Comment:
should we add a simple test file for testing indents?

Reply:
sure, but I'm not very familiar with testing indents. can you link an example?

Reply:
It should be sufficient to add a file like this https://github.com/nvim-treesitter/nvim-treesitter/blob/588cdb6c0652356dacc7d29bb4bb2b273df2df86/tests/indent/css_spec.lua#L1-L22 and put a example `ron` file in a subfolder. The `new line:` section is optional and allows you to test how newly inserted lines in a file should be indented.

Reply:
@amaanq this is not strictly necessary. We can also merge as is.

Reply:
> For this config block, you probably need to add `ft = "ron"` as Neovim doesn't recognize this file type natively. https://github.com/nvim-treesitter/nvim-treesitter/blob/588cdb6c0652356dacc7d29bb4bb2b273df2df86/tests/indent/css_spec.lua#L7

Yeah I've submitted a PR to vim to add that, should be merged soon :)

Reply:
> @amaanq this is not strictly necessary. We can also merge as is.

Yeah that's fine, I'll add a test shortly either way!